src/transform/headings.ts

```diff
@@ -78,7 +81,7 @@ function getFlatListHeadings(tokens: Token[]) {
         }
 
         headings.push({
-            title: getTitle(tokens[i + 1]),
+            title: href + getTitle(tokens[i + 1]),
```

Comment:
why title need href ?

Reply:
This is a mistake. I moved to href block